src/components/Login.tsx

```diff
@@ -8,14 +8,14 @@ import { fetchToken } from '../services/fetch-token'
 import { auth as firebaseuiAuth } from 'firebaseui'
 import { isAuthEmulatorEnabled } from '../utils/emulators'
 import { JWTToken } from '../../types/JWT'
-import { selectToken } from '../utils/selectors'
+import { selectIsPendingToken, selectToken } from '../utils/selectors'
 import { useDispatch, useSelector } from 'react-redux'
 
 const uiConfig = {
     // Popup signin flow rather than redirect flow.
     signInFlow: 'popup',
     // Redirect to /signedIn after sign in is successful. Alternatively you can provide a callbacks.signInSuccess function.
-    signInSuccessUrl: '/',
+    signInSuccessUrl: '/admin',
```

Comment:
different case by user type here

Reply:
I did it related to this
https://github.com/Instant-Visio/InstantVisio-WebApp/pull/330#discussion_r542139282